test/Microsoft.AspNet.Mvc.ModelBinding.Test/Metadata/DefaultModelMetadataProviderTest.cs

```diff
@@ -0,0 +1,209 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Testing;
+using System;
+using System.Linq;
+using System.Reflection;
+using Xunit;
+
+namespace Microsoft.AspNet.Mvc.ModelBinding.Metadata
+{
+    public class DefaultModelMetadataProviderTest
+    {
+        [Fact]
+        public void GetMetadataForType_IncludesAttributes()
+        {
+            // Arrange
+            var provider = new EmptyModelMetadataProvider();
```

Comment:
Instead of doing this can we create a helper method which constructs ```DefaultModelMetadataProvider```  object exactly like ```EmptyModelMetadataProvider``` does? Its not clear that we are testing ```DefaultModelMetadataProvider``` when i look at it first.